src/index.js

```diff
@@ -58,10 +69,30 @@ const renderDoc = async function renderDocs(path) {
   };
 };
 
+const renderFragment = async function renderFragments(path) {
```

Comment:
In our case, we will need to use fetch as these files would not be accessed locally, instead it will be read from a source location. Currently, we are storing these files in github temporarily but later these content will be served from AEM UE or some other source. So, we won't be able to make use of fs as it is trying to find the file in the directory.